doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex

```diff
@@ -4062,6 +4062,7 @@ \subsubsection{Inputs}\label{inputs-19-001}
 
 \paragraph{Field: Minimum Outdoor Dry-Bulb Temperature for Compressor Operation}\label{field-minimum-outdoor-dry-bulb-temperature-for-compressor-operation}
 
+This numeric field defines the minimum outdoor air dry-bulb temperature where the heating coil compressor turns off. If this input field is left blank, the default value is -8 °C.
```

Comment:
I haven't verified the -8C, but this change in general seems totally fine.

Reply:
@Myoldmopar this field all DX heating coils have -8.0C as the minimum value while all DX cooling coils have -25.0C based on the latest idd.

Reply:
Thanks for confirming @Nigusse